alma/alma-8.x/alma-8.7-hpc/install.sh

```diff
@@ -4,15 +4,24 @@ set -ex
 # set properties
 source ./set_properties.sh
 
+# install spack
+$ALMA_COMMON_DIR/install_spack.sh
+# Activate the environment/ container
+source /etc/profile
+export PATH="/sbin:/bin:/usr/sbin:/usr/bin:$PATH"
+
+# install compilers
+./install_gcc.sh
```

Comment:
should this be moved above spack, since spack needs it as well? Or are we ok with the scripts being independent and spack installing gcc, with some of this being redundant? Is spack needed for the gcc install?

Reply:
Just noticed spack is used in the gcc install. So entirely possible some stuff will be repeated

Reply:
Yes, that is correct. Spack heavily uses gcc for installation of packages